test/integration/stats_integration_test.cc

```diff
@@ -129,5 +136,65 @@ TEST_P(StatsIntegrationTest, WithTagSpecifierWithFixedValue) {
   EXPECT_EQ(live->tags()[0].value_, "xxx");
 }
 
+// TODO(cmluciano) Refactor once envoyproxy/envoy#5624 is solved
+// TODO(cmluciano) Add options to measure multiple workers & without stats
+class ClusterMemoryTest : public BaseIntegrationTest {
```

Comment:
nit: comment that no tests are made, and that this just a helper class for instantiating in other tests.  WDYT of calling this ClusterMemoryTestHelper rather than ClusterMemoryTest to highlight that this is not a normal test-class?